tools/dataform_assertions/definitions/tests/date_assertions_unit_test.sqlx

```diff
@@ -0,0 +1,40 @@
+config {
+    type: "view",
+    schema: "dev_data_zone",
+    name: "date_assertions_test",
+    assertions: {
+        nonNull: ["test_date"],
+        rowConditions: 
+        [
+            date_assertions.test_future_date("test_date"),
+            date_assertions.test_valid_years("test_date"),
+            date_assertions.test_date_format("test_date", "yyyymmdd")
```

Comment:
looks like you only test for yyyymmdd format. your assertion logic has an if else to check for other formats. what's the intended behavior?